keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -65,7 +65,7 @@
             }
           },
           "default": {
-            "description": "Default response. It will be deserialized as per the error definition specified in the schema. An exception will be thrown.",
+            "description": "Default response. It will be deserialized per the error definition specified in the schema. An exception will be thrown.",
```

Comment:
@BrucePerlerMS, yep, documenting through Swagger is new to most of us. For default values in models, the default property of the [swagger schemaObject](http://swagger.io/specification/#schemaObject) should be used to annotate the value if it is contractual (i.e. it will not change on the service unless there is a corresponding API version change). Specifying it there will allow tools determine the best way to present the information to the user, which is not possible (or at least very difficult) if you put the values in the English text description.